### ISSUE TITLE
Finalize Operators (and reserved symbols)

### ISSUE THREAD

**Issue by TheOtherZech** [TheOtherZech/vada#4]
TL;DR: There are more operators in the Tour than currently implemented, and the operators need a solid design pass focusing on disambiguation and semantics. Vada has a **lot** of operators, and I want to avoid getting to esoteric here.

The Tour formally introduces:

| Op Group                     | Binding Power | Comments                                            |
| ------------------------ | ------------- | --------------------------------------------------- |
| `~>`                              | (0, 0)        | Outer Pipe                                          |
| `, ;`                                | (1, 2)        | Sequence Delineators                                |
| `=`                                | (3, 4)        | Alias Bind                                          |
| `\|`          | (5, 6)        | Disjunction                                         |
| `&`                                | (7, 8)        | Conjunction                                         |
| `> >= < <= == !=`      | (9, 10)       | Equality Operators                                  |
| `+ -`                              | (11, 12)      | Standard Math                                       |
| `* / ^ %`                        | (13, 14)      | Standard Math                                       |
| `** // ^^ %% !!`             | (13, 14)      | Spicy Math                                          | 
| `.& .&#124; .^ .~ .< .>` | (15, 16)      | Bitwise Ops                                         |
| ...                                   | ...           | Where the lower unary operators fit in power-wise   |
| `-> ?>`                          | (19, 20)      | Inner Pipe and If Pipe                              |
| `.. ..= =..`                        | (21, 22)      | Ranges expressions are effectively operands.        |
| ...                                   | ...           | Signs, Inverse, and Transpose prefixes              |
| `. ::`                                | (∞, ∞)        | Accessors are always the tightest-binding operators  |

| Op Group                | Binding Power | Comments          |
| ----------------------- | ------------- | ----------------- |
| `> >= < <=`             | (0, 17)       | Value Constraints |
| `* // ^ `               | (0, 17)       | Value Constraints |
| `~= !=`                 | (0, 17)       | Composition Constraints|
| `.. ..=`                | (0, 17)       | Upper Range       |
| `...`                   | (0, 17)       | Spread Op         |
| `+ - ~ %`               | (0, 23)       | Signs, Inverse, Transpose |

| Op Group | Binding Power | Comments    |
| -------- | ------------- | ----------- |
| `.. =..` | (17, 0)       | Lower Range |

And also informally introduces these uniform ops:

* prefix inversion `~someVec`
* prefix transposition `%someVec` 
* infix wedge product `someVec ^^ anotherVec`
*  infix cross product `someVec ** anotherVec`

At a bare minimum, prefix inversion will need to be changed.
